src/Model/Behavior/FileStorageBehavior.php

```diff
@@ -98,12 +99,12 @@ public function beforeMarshal(Event $event, ArrayAccess $data): void
      * @param \ArrayObject $options The options for the query
      * @return void
      */
-    public function beforeSave(EventInterface $event, EntityInterface $entity, ArrayObject $options): void
+    public function beforeSave(EventInterface $event, EntityInterface $entity, ArrayObject $options): bool
     {
         if (!$this->_isFileUploadPresent($entity)) {
             $event->stopPropagation();
 
-            return;
+            return false;
```

Comment:
Is this actually relevant? AFAIR Cake didn't bother about any return value from a callback like beforeSave()?

Reply:
@skie any answer to this?